cypress/support/fragments/eholdings/eHoldingsTitlesSearch.js

```diff
@@ -37,5 +38,25 @@ export default {
     cy.do(selectionStatusAccordion
       .find(RadioButton(selectionStatus)).click());
     eHoldingsTitles.waitLoading();
-  }
+  },
+  openTitle(itemTitle) {
+    cy.do([
+      searchResults.find(Heading(itemTitle)).click(),
+    ]);
+  },
+  checkTitleInfo(publicationType, titleValue) {
+    cy.expect([
+      titleInfoPane.find(KeyValue({ value: including(titleValue) })).exists(),
+      titleInfoPane.find(KeyValue({ value : publicationType })).exists(),
+    ]);
+  },
+  getViaApi(query) {
+    return cy.okapiRequest({
+      path: 'eholdings/titles',
+      searchParams: query,
+      isDefaultSearchParamsRequired: false,
+    }).then((res) => {
+      return res.body.data[0];
```

Comment:
res.body.data[0] this is not good way for getting data.
What if anyone would get more than one title?
I mean better to return array instead of first value from array.
res.body.data
Inside test you can get first value

Reply:
changed